### PR TITLE
add module3

### PR DESCRIPTION
**Pull request by ins21**

### CODE REVIEW

File: module3/config/db_config.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export const db_config = {
+  name: 'users_db',
+  user: 'postgres',
+  password: '123456',
```
Comment (Vladlan) [inline module3/config/db_config.js]: Store password in a git folder is really bad practice. Use .env files for creds (https://www.npmjs.com/package/dotenv).

File: module3/sql/db.sql [hunk 2]
```diff
@@ -0,0 +1,17 @@
+CREATE DATABASE users_db;
```
Comment (Vladlan) [inline module3/sql/db.sql]: Add migration logic to fill other hosts with this data on start.

File: module3/validation/validation.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+const schema = Joi.object().keys({
+  login: Joi.string().alphanum().min(1).required(),
+  password: Joi.string().required().pattern(new RegExp('[A-Za-z0-9]*([a-zA-Z]+[0-9]+|[0-9]+[a-zA-Z]+)')),
+  age: Joi.number().min(4).max(130).required()
+});
+
+export const validator = (req, res, next) => {
+  const { error } = schema.validate(req.body);
+  if (error) return res.status(400).json({ error: error.details[0].message });
```
Comment (Vladlan) [inline module3/validation/validation.js:11]: How about to use http-status-codes?

**Review by Vladlan — CHANGES_REQUESTED** [review]

File: module3/config/db_config.js (hunk 1 above)
Comment (ins21) [reply module3/config/db_config.js]: Thanks, fixed that.

File: module3/sql/db.sql (hunk 2 above)
Comment (ins21) [reply module3/sql/db.sql]: Done

File: module3/validation/validation.js (hunk 3 above)
Comment (ins21) [reply module3/validation/validation.js:11]: I would suggest to leave it as is, if you don't mind)

**Review by Vladlan — APPROVED** [review]

**Pull request merged**